syntax/kitty.vim

```diff
@@ -9,7 +9,7 @@ syn match kittyNumber '\s[+-]\?\d\+\.\?\d*\(%\|px\|pt\|em\)\?'ms=s+1 contained c
 syn match kittyUnit '\(px\|pt\|em\)' contained
 syn match kittyKW '^\s*\S*' contains=kittyKeyword,kittyInvalidKeyword nextgroup=kittySt
 syn match kittyComment /^\s*#.*$/ contains=kittyTodo
-syn match kittyInclude /^\s*include/ display
+syn match kittyInclude /^\s*\(env\|glob\)?include/ display
```

Comment:
Oops, the question mark is not escaped.

Reply:
True! Thanks for flagging it.
Fixed at [2fa2bfb](https://github.com/fladson/vim-kitty/commit/2fa2bfbed408904dc7b31909a82c2a99c7a9174b)